flink-python/dev/lint-python.sh

```diff
@@ -213,11 +213,12 @@ function install_tox() {
         fi
     fi
 
-    # virtualenv 16.6.2 released in 2019-07-14 is incompatible with py27 and py34,
-    # force install an older version(16.0.0) to avoid this problem.
-    $CONDA_PATH install -p $CONDA_HOME -c conda-forge virtualenv=16.0.0 tox -y -q 2>&1 >/dev/null
+    # tox 3.14 released in conda-forge will download 0.1x and 0.2x versions of importlib_metadata
```

Comment:
What about change the comments as follows?
`# tox 3.14 depends on both 0.19 and 0.23 of importlib_metadata at the same time and conda will try to install both these two versions and it will cause problems occasionly. Using pip as the package manager could avoid this problem.`